builds/Makefile

```diff
@@ -12,12 +12,23 @@ ifdef ENABLE_XAMARIN
 XAMARIN_AUTOGEN_FLAGS=--enable-extension-module=xamarin
 XAMARIN_CONFIGURE_FLAGS=--enable-extension-module=xamarin
 -include $(MACCORE_PATH)/builds/Makefile.include
+else
+XAMARIN_CONFIGURE_FLAGS=--enable-extension-module
```

Comment:
This doesn't look right, you shouldn't pass `--enable-extension-module` unless ENABLE_XAMARIN is defined.